platform/ui/src/components/TableRow/TableRow.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const TableRow = ({ children, className, isTableHead, style }) => {
+  return (
+    <div className={classnames('w-full flex', className)} style={style}>
+      {React.Children.map(children, child =>
+        React.cloneElement(child, { isTableHead })
```

Comment:
If we're only supporting "text" or raw html for now, is there any need to expose `TableCell`?

Reply:
I've created a condition to handle how the children should be rendered. If it receives a string or a custom content, it will not break the application. Also, a custom propTypes method was created in the props definition to display customized messages depending on the element type of children.

The children of a `TableRow` is normally a `TableCell`. That's what we expect. It can even receive a `div`, a `string` or custom content and it will work, but if it receives a string or an invalid element, it will throw a console warning:

![image](https://user-images.githubusercontent.com/1905961/77477306-e5a0e580-6dfa-11ea-86d2-ee6008cc09bc.png)

Reply:
Our current usage only has Bold text headers and regular text inside of each cell. Is it worth creating a component for that? That's mostly what I'm getting at here.

We may eventually want something more complex, but today, a simple table just shows text tabular data with some light styling.

Reply:
Basically, the type checking and display logic adds cases we won't be using out of the gate. If it's not something our current design needs, it's extra weight we need to test/support that is more likely to break.

Reply:
So, this is basically what `TableCell` currently does:

- `colSpan` prop to define its size. (I just added)
- It adds a blue `border-right` to separate the cell (if it's in the TableBody tree).
- `align` prop, to define its content alignment.
- base classes (Tailwind) that set a default padding and word-break.

Even being simple, I think that's enough to have an exposed `TableCell` component. And also to keep consistency in the use of our table components, even if they are simple, to avoid things like:

```javascript
<TableRow>
  <div className="tailwind-classes">...</div>
  <div className="tailwind-classes">...</div>
  <div className="tailwind-classes">...</div>
</TableRow>
```
About the type checking and display logic, I can definitely remove them, as `TableRow` will naturally/always expect a `TableCell` or a `<div>`, but never a string.

I'm sorry for this long text 😅 we can discuss offline if you want.